modules/logging-forwarding-azure.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// logging/configuring-log-forwarding.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="<logging-forwarding-azure>_{context}"]
+= Forwarding to Azure Monitor Logs
```

Comment:
@libander Vale claims it should be "Microsoft Azure" instead of "Azure", and calls instances of only "Azure" an error. I won't litter the review with all that error text, but simply bring it to your attention.  When I looked elsewhere in the docs, I found only "Azure" in **_titles_** and "Microsoft Azure" in **_text_**. And even that rule, inconsistently.